tests/run-pass/communication.rs

```diff
@@ -0,0 +1,6 @@
+// ignore-windows: TODO env var emulation stubbed out on Windows
+// compile-flags: -Zmiri-enable-communication
+
+fn main() {
+    assert!(std::env::var("PWD").is_ok());
```

Comment:
I am not sure which variable is appropiate here. I suppose that something like POSIX should specify which env variables must always be defined. On the other hand we could use `RUSTFLAGS` or another variable exported in the `miri` script

Reply:
not sure if compiletest-rs can set env vars for a single test via comments. If so, do that. If not, set it via the `compiletest.rs` file. Invent a new env var like `MIRI_ENV_VAR_TEST`